data/lhc-hxswg/makeSMtables.py

```
@@ -202,22 +217,68 @@ def col2num(col_str):
     except AttributeError:
         pass
+    # want to have at most 4 decimal digits.
+    v = round(v, 4)
     return "{:+}".format(v)
 
 
 def main(o):
-    f = xlrd.open_workbook(o.input)
+    try:
+        f = xlrd.open_workbook(o.input) 
+    except:
```

Ruff (0.15.15)

[error] Do not use bare `except`

(E722)